### PR TITLE
Domains: Add link to change email when domain is pending verification

### PR DESCRIPTION
**Pull request by eltongo**
#### Changes proposed in this Pull Request

This PR adds a link to change the email address when the domain is pending verification.

#### Testing instructions

Change the email address for a domain you own. You should see something similar to the screenshot below:

Screenshot 2020-05-06 at 3 50 21 PM

Make sure clicking the link sends you to the contact info form.

### CODE REVIEW

File: client/my-sites/domains/domain-management/components/email-verification/index.jsx [hunk 1]
```diff
@@ -134,15 +134,20 @@ class EmailVerificationCard extends React.Component {
...
 				<p>{ this.props.verificationExplanation }</p>
-				<Button busy={ submitting } disabled={ submitting } onClick={ this.handleSubmit }>
-					{ submitting ? translate( 'Sending…' ) : translate( 'Resend email' ) }
-				</Button>
+				<div className="email-verification__actions">
+					<Button busy={ submitting } disabled={ submitting } onClick={ this.handleSubmit }>
+						{ submitting ? translate( 'Sending…' ) : translate( 'Resend email' ) }
+					</Button>
+					{ changeEmailHref && (
+						<a href={ changeEmailHref }>{ translate( 'Change your email address' ) }</a>
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/components/email-verification/index.jsx:148]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Change Your Account Email Address' )` *ES Score: 6*

**Comment by jancavan** [comment]
Thanks @fditrapani and @eltongo. 

I just wanted to point out the copy:
> We sent contact@email.com an email to verify your contact information...

Not sure if it's just me, but the way it's written, it feels like it sent an email to a different person that's not the user?

Maybe something like this?

> We sent you an email to contact@email.com to verify your contact information...

**Comment by eltongo** [comment]
Alright, I did another update. @jancavan I updated the copy you were referring to, since the new copy feels better.

Also, @fditrapani the contact form should now send you back to the domain settings when clicking the link :)

File: client/my-sites/domains/domain-management/components/icann-verification/index.jsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ class IcannVerificationCard extends React.Component {
 
 		if ( 'new-status' === explanationContext ) {
 			return translate(
-				'We sent {{strong}}%(email)s{{/strong}} an email to verify your contact information. Please complete the verification or your domain will stop working in {{strong}}10 days{{/strong}}.',
+				'We sent you an email at {{strong}}%(email)s{{/strong}} to verify your contact information. Please complete the verification or your domain will stop working in {{strong}}10 days{{/strong}}.',
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/components/icann-verification/index.jsx:35]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Pull request merged**